lib/ansible/modules/remote_management/oneview/oneview_datacenter.py

```diff
@@ -122,19 +121,16 @@
 
 
 class DatacenterModule(OneViewModuleBase):
-    MSG_CREATED = 'Data Center added successfully.'
-    MSG_UPDATED = 'Data Center updated successfully.'
-    MSG_ALREADY_PRESENT = 'Data Center is already present.'
-    MSG_DELETED = 'Data Center removed successfully.'
-    MSG_ALREADY_ABSENT = 'Data Center is already absent.'
+    MSG_CREATED = 'Datacenter added successfully.'
+    MSG_UPDATED = 'Datacenter updated successfully.'
+    MSG_ALREADY_PRESENT = 'Datacenter is already present.'
+    MSG_DELETED = 'Datacenter removed successfully.'
+    MSG_ALREADY_ABSENT = 'Datacenter is already absent.'
     MSG_RACK_NOT_FOUND = 'Rack was not found.'
     RESOURCE_FACT_NAME = 'datacenter'
 
     argument_spec = dict(
-        state=dict(
-            required=True,
-            choices=['present', 'absent']
-        ),
+        state=dict(type='str', default='present', choices=['present', 'absent']),
         data=dict(required=True, type='dict')
```

Comment:
To make the argument_spec easier to read, a good idea might be to always place the `type` first, followed by `default` or `required` and then the rest, such as suggested in [this comment](https://github.com/ansible/ansible/pull/28336#discussion_r134806508).
The `choices` could also be sorted